firecracker-control/local.go

```diff
@@ -167,7 +167,10 @@ func (s *local) CreateVM(requestCtx context.Context, req *proto.CreateVMRequest)
 
 	defer func() {
 		if err != nil {
-			cmd.Process.Kill()
+			err = cmd.Process.Kill()
+			if err != nil {
```

Comment:
nit:

```go
if err := cmd.Process.Kill(); err != nil {
}
```

This also doesn't stomp over the `err` variable in the function scope.